forem/forem
# Fixes forem-docs #127: References to rtx are out-of-date

## What type of PR is this? (check all applicable)


## Description
This PR
- renames the config file `.rtx.toml` to `.mise.toml`
- updates references to `rtx` to `mise`

## Related Tickets & Documents

- Related Issue: https://github.com/forem/forem-docs/issues/127

## QA Instructions, Screenshots, Recordings

Verify this change works as expected by running `mise install` or `mise doctor` from this branch.

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] No, and this is why: documentation update

## [optional] Are there any post deployment tasks we need to perform?
I didn't find a script that runs `rtx install` for deployment, so I don't believe so.
